src/get-started/codelab.md

```diff
@@ -613,57 +607,49 @@ lazily, on demand.
     ```diff
     --- step3_stateful_widget/lib/main.dart
     +++ step4_infinite_list/lib/main.dart
-    @@ -13,27 +13,43 @@
-       const MyApp({Key? key}) : super(key: key);
+    @@ -14,12 +14,12 @@
 
        @override
        Widget build(BuildContext context) {
-    -    return MaterialApp(
+         return MaterialApp(
     -      title: 'Welcome to Flutter',
-    -      home: Scaffold(
-    -        appBar: AppBar(
-    -          title: const Text('Welcome to Flutter'),
-    -        ),
-    -        body: const Center(
-    -          child: RandomWords(),
-    -        ),
-    -      ),
-    +    return const MaterialApp(
     +      title: 'Startup Name Generator',
-    +      home: RandomWords(),
-         );
+           home: Scaffold(
+             appBar: AppBar(
+    -          title: const Text('Welcome to Flutter'),
+    +          title: const Text('Startup Name Generator'),
+             ),
+             body: const Center(
+               child: RandomWords(),
+             ),
+    @@ -28,12 +28,30 @@
        }
      }
 
      class _RandomWordsState extends State<RandomWords> {
     +  final _suggestions = <WordPair>[];
-    +  final _biggerFont = const TextStyle(fontSize: 18.0);
+    +  final _biggerFont = const TextStyle(fontSize: 18);
     +
        @override
        Widget build(BuildContext context) {
     -    final wordPair = WordPair.random();
     -    return Text(wordPair.asPascalCase);
-    +    return Scaffold(
```

Comment:
Because I delayed moving the Scaffold until a later step of the codelab, there probably needs to be an update to the surrounding codelab text to reflect that this change.

Reply:
Taking a look now :)

Reply:
Step 5 (the last one of this part) is a bit empty now, only changing the title of the app bar.

I think for now, this is fine, but perhaps followup work should consider moving that an earlier step?

Reply:
Sounds good